scraper.go

```diff
@@ -16,14 +17,21 @@ const (
 	OpUpdateTarget  = "UpdateTarget"
 )
 
+var (
+	DefaultScraperInterval = *options.MustParseDuration("10s")
+	DefaultScraperTimeout  = *options.MustParseDuration("30s")
```

Comment:
the default value base on previous usage:
https://github.com/influxdata/influxdb/blob/a64c4fd138ef11336de84b4bd42bc0f49b7d8d1d/cmd/influxd/launcher/launcher.go#L587-L591